src/WinRT.Runtime/Projections/IPropertyValue.netstandard2.0.cs

```diff
@@ -327,12 +327,46 @@ private static T CoerceValue<T>(object value)
                 {
                     return (T)(object)guid.ToString("D", global::System.Globalization.CultureInfo.InvariantCulture);
                 }
+                else if (typeof(T) == typeof(byte))
```

Comment:
We can probably avoid this change, we don't really care about NS2.0 anyway (it doesn't support trimming).

Reply:
Honestly I like the change for the consistency, as the two implementations should be the same anyway.